src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Note that the last star doesn't need marigin (try something with `:not `and `:last-child` pseudo classes)